contracts/modules/cover/Cover.sol

```diff
@@ -416,7 +444,7 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon {
     );
   }
 
-  function performPayoutBurn(
+  function performStakeBurn(
```

Comment:
What's the rationale behind this rename? Isn't "burnAmount" supposed to be the amount paid out (i.e. ETH/DAI, not NXM)? It definitely isn't the stake (in NXM) that is burned and we don't necessarily burn NXM when we make a payout, that's just a consequence of the payout.

Reply:
Discussed off-band.